Assets/MRTK/SDK/Features/UX/Materials/Teleport/License.txt

```diff
@@ -0,0 +1,81 @@
+-------------------------------------------------------------------------------
+MRTKExtensionForOculusQuest
```

Comment:
This seems to have the old license for MRTK-Quest.
Can you update it to the new one?
https://github.com/provencher/MRTK-Quest/blob/master/License.txt

Reply:
Looks like one License.txt was updated but this one still has the old verbiage